draft-ietf-quic-recovery.md

```diff
@@ -438,11 +438,13 @@ and larger thresholds increase loss detection delay.
 A Probe Timeout (PTO) triggers sending one or two probe datagrams when
 ack-eliciting packets are not acknowledged within the expected period of
 time or the handshake has not been completed.  A PTO enables a connection to
-recover from loss of tail packets or acknowledgements. The PTO algorithm used
-in QUIC implements the reliability functions of Tail Loss Probe
-{{?TLP=I-D.dukkipati-tcpm-tcp-loss-probe}} {{?RACK}}, RTO {{?RFC5681}} and
-F-RTO algorithms for TCP {{?RFC5682}}, and the timeout computation is based on
-TCP's retransmission timeout period {{?RFC6298}}.
+recover from loss of tail packets or acknowledgements.  As with loss
+detection, the probe timeout is per packet number space.
+
+The PTO algorithm used in QUIC implements the reliability functions of
+Tail Loss Probe {{?TLP=I-D.dukkipati-tcpm-tcp-loss-probe}} {{?RACK}},
+RTO {{?RFC5681}} and F-RTO algorithms for TCP {{?RFC5682}}, and the timeout
```

Comment:
The commas here are odd.  I think that you could start a new sentence here:

```suggestion
RTO {{?RFC5681}}, and F-RTO  {{?RFC5682}} algorithms for TCP.  The timeout
```

Yeah, this is old text, but it's really confusing still.